test/VSSDK.TestExtension/MEF/TextviewCreationListener.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using Community.VisualStudio.Toolkit.Shared;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+
+namespace TestExtension.MEF
+{
+    [Export(typeof(IWpfTextViewCreationListener))]
+    [ContentType("text")]
+    [TextViewRole(PredefinedTextViewRoles.PrimaryDocument)]
+    internal class TextviewCreationListener : IWpfTextViewCreationListener, IDisposable
```

Comment:
nit: shouldn't there be a capital V here?
```diff
- internal class TextviewCreationListener : IWpfTextViewCreationListener, IDisposable
+ internal class TextViewCreationListener : IWpfTextViewCreationListener, IDisposable
```

Reply:
Thanks, have sent a new PR.